scripts/xbstrap

```diff
@@ -1816,10 +1917,11 @@ try:
 		do_list_tools(args)
 	elif args.command == 'list-pkgs':
 		do_list_pkgs(args)
+	elif args.command == 'run-pkg':
```

Comment:
I still think that we should change the syntax from `run-pkg <package> <target>` to `run --pkg <package> <target>` to keep the door open for global run targets that can be added in a future patch.

Reply:
Addressed in 77adabe.